### ISSUE TITLE
Creation of redshift subnet group is failing using terraform

### ISSUE THREAD

**Issue by arussel** [localstack/localstack#3440]
# Type of request: This is a ...

[x] bug report
[ ] feature request

# Detailed description
I have created a very small project to reproduce the issue: https://github.com/arussel/terraform-localstack-redshift-subnet-group
3 Resources are created: aws_vpc, aws_subnet, aws_redshift_subnet_group

## Expected behavior
When running against AWS, the result is:
```
aws_vpc.redshift_vpc: Creating...
aws_vpc.redshift_vpc: Still creating... [10s elapsed]
aws_vpc.redshift_vpc: Still creating... [20s elapsed]
aws_vpc.redshift_vpc: Still creating... [30s elapsed]
aws_vpc.redshift_vpc: Still creating... [40s elapsed]
aws_vpc.redshift_vpc: Still creating... [50s elapsed]
aws_vpc.redshift_vpc: Still creating... [1m0s elapsed]
aws_vpc.redshift_vpc: Still creating... [1m10s elapsed]
aws_vpc.redshift_vpc: Still creating... [1m20s elapsed]
aws_vpc.redshift_vpc: Still creating... [1m30s elapsed]
aws_vpc.redshift_vpc: Still creating... [1m40s elapsed]
aws_vpc.redshift_vpc: Still creating... [1m50s elapsed]
aws_vpc.redshift_vpc: Still creating... [2m0s elapsed]
aws_vpc.redshift_vpc: Creation complete after 2m9s [id=vpc-0498129c1014fa82b]
aws_subnet.redshif_subnet: Creating...
aws_subnet.redshif_subnet: Still creating... [10s elapsed]
aws_subnet.redshif_subnet: Still creating... [20s elapsed]
aws_subnet.redshif_subnet: Still creating... [30s elapsed]
aws_subnet.redshif_subnet: Creation complete after 32s [id=subnet-074aef05bacbe3177]
aws_redshift_subnet_group.redshift_subnet_group: Creating...
aws_redshift_subnet_group.redshift_subnet_group: Still creating... [10s elapsed]
aws_redshift_subnet_group.redshift_subnet_group: Still creating... [20s elapsed]
aws_redshift_subnet_group.redshift_subnet_group: Creation complete after 22s [id=my-redshift-subnet-group]

Apply complete! Resources: 3 added, 0 changed, 0 destroyed.
```

## Actual behavior
When running agains Localtack:
```
aws_vpc.redshift_vpc: Creating...
aws_vpc.redshift_vpc: Creation complete after 2s [id=vpc-5e2812d9]
aws_subnet.redshif_subnet: Creating...
aws_subnet.redshif_subnet: Creation complete after 0s [id=subnet-e7a1bef3]
aws_redshift_subnet_group.redshift_subnet_group: Creating...

Error: Unable to find Redshift Subnet Group: []*redshift.ClusterSubnetGroup(nil)
```

# Steps to reproduce

## Command used to start LocalStack
A docker compose file using localstack/localstack as image (see project)

## Client code (AWS SDK code snippet, or sequence of "awslocal" commands)
Terraform, see the main.tf provided. Note again that the code is working against AWS.

**Comment by whummer** [comment]
Thanks for reporting @arussel . This should be fixed in 218f09e - can you please pull the `latest` Docker image and give it another try? Please report here if the problem persists. Thanks

**Comment by arussel** [comment]
@whummer  confirmed this is fixed with docker image b52081af2988
